### ISSUE TITLE
BUG: Unhandled error in plugin discover: 'Season' object has no attribute 'number'

### ISSUE THREAD

**Issue by programatix** [Flexget/Flexget#1956]
### Expected behaviour:
There was no error until I upgraded to version 2.10.83. I'm not sure what version I was running previously before I upgrade, but so far I never encountered this error. No change was made to the list of "anime" in my config.yml.

From log, it seems that 'next_series_episodes' found the 'season' info for 'Fate Apocrypha' from somewhere, causing the error.

### Actual behaviour:
It crashed with a "BUG" status

### Steps to reproduce:
- Step 1: Execute task 'download-anime-discover'

#### Config:
```
templates:
  cartoon-metainfo:
    thetvdb_lookup: yes

  cartoons:
    series:
      settings:
        cartoon:
          propers: 3 days
          quality: <=720p
          set:
            path: '/{? folder.root ?}{? folder.cartoon ?}{{series_name}}'
        cartoon_anime:
          propers: 3 days
          quality: <=720p
          identified_by: sequence
          from_group: horriblesubs
          set:
            path: '/{? folder.root ?}{? folder.cartoon ?}{{series_name}}'
        anime_various:
          propers: 3 days
          quality: <=720p
          identified_by: sequence
          tracking: backfill
          from_group: horriblesubs
          set:
            path: '/{? folder.root ?}{? folder.various ?}{{series_name}}'
      cartoon_anime:
        - Hunter X Hunter
        - World Trigger
        - "Mobile Suit Gundam - Iron-Blooded Orphans"
        - Boku no Hero Academia
      anime_various:
        - Sagrada Reset
        - "Knight's & Magic"
        - Fate Apocrypha:
            from_group: UTW
        - Shingeki no Bahamut - Virgin Soul
        - Busou Shoujo Machiavellianism
        - 'Re-Creators'
        - Granblue Fantasy The Animation
        - Shingeki no Kyojin S2:
            begin: 26
        - 'Ao no Exorcist - Kyoto Fujouou-hen'
        - Mobile Suit Gundam Unicorn RE 0096
        - "Gundam Reconguista in G"
        - Macross Delta:
            from_group: gg
    template: transmissionRPC

tasks:
  download-anime-discover:
    template:
      - cartoon-metainfo
      - cartoons
      - notifyViaEmail
    discover:
      what:
        - next_series_episodes:
            backfill: yes
            from_start: yes
      from:
        - horriblesubs: yes
        - nyaa:
            category: anime eng
            filter: trusted only
      release_estimations: ignore
      interval: 1 hour
```
  
#### Log:
```
2017-09-03 10:07 TRACE    next_series_episodes download-anime-discover Evaluating episodes for series Gundam Reconguista in G, season 0
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Gundam Reconguista in G` with parameters season: 0, downloaded: False
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Episode(id=1375,identifier=26,season=0,number=26)> to None
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Episode(id=1375,identifier=26,season=0,number=26)> to None
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Episode(id=1375,identifier=26,season=0,number=26)> and <Episode(id=3207,identifier=1,season=0,number=1)>
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: title = 'Gundam Reconguista in G 027'
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: original_url = ''
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: url = ''
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: search_strings = ['Gundam Reconguista in G 027', 'Gundam Reconguista in G 27']
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_name = 'Gundam Reconguista in G'
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_alternate_names = []
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_season = 0
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_episode = 27
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: season_pack_lookup = False
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_id = 27
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_id_type = 'sequence'
2017-09-03 10:07 TRACE    next_series_episodes download-anime-discover evaluating Granblue Fantasy The Animation
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Granblue Fantasy The Animation` with parameters season: None, downloaded: False
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Episode(id=4426,identifier=13,season=0,number=13)> to None
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Episode(id=4426,identifier=13,season=0,number=13)> to None
2017-09-03 10:07 TRACE    next_series_episodes download-anime-discover Evaluating episodes for series Granblue Fantasy The Animation, season 0
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Granblue Fantasy The Animation` with parameters season: 0, downloaded: False
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Episode(id=4426,identifier=13,season=0,number=13)> to None
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Episode(id=4426,identifier=13,season=0,number=13)> to None
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: title = 'Granblue Fantasy The Animation 14'
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: original_url = ''
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: url = ''
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: search_strings = ['Granblue Fantasy The Animation 14', 'Granblue Fantasy The Animation 014']
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_name = 'Granblue Fantasy The Animation'
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_alternate_names = []
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_season = 0
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_episode = 14
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: season_pack_lookup = False
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_id = 14
2017-09-03 10:07 TRACE    entry         download-anime-discover ENTRY SET: series_id_type = 'sequence'
2017-09-03 10:07 TRACE    next_series_episodes download-anime-discover evaluating Fate Apocrypha
2017-09-03 10:07 ERROR    series        download-anime-discover cannot compare season when identifier is not 'ep'
2017-09-03 10:07 TRACE    series        download-anime-discover checking if 11 is smaller than 0
2017-09-03 10:07 TRACE    next_series_episodes download-anime-discover Evaluating episodes for series Fate Apocrypha, season 11
2017-09-03 10:07 DEBUG    series        download-anime-discover no episodes found for series `Fate Apocrypha` with parameters season: 11, downloaded: False
2017-09-03 10:07 TRACE    series        download-anime-discover comparing <Season(id=2,identifier=S11,season=11,completed=False)> to None
2017-09-03 10:07 CRITICAL task          download-anime-discover BUG: Unhandled error in plugin discover: 'Season' object has no attribute 'number'
2017-09-03 10:07 CRITICAL manager       download-anime-discover An unexpected crash has occurred. Writing crash report to /opt/etc/flexget/crash_report.2017.09.03.100729681582.log. Please verify you are running the latest version of flexget by using "flexget -V" from CLI or by using version_checker plugin at http://flexget.com/wiki/Plugins/version_checker. You are currently using version 2.10.83
2017-09-03 10:07 DEBUG    manager       download-anime-discover Traceback:
Traceback (most recent call last):
  File "/opt/lib/python3.6/site-packages/flexget/task.py", line 486, in __run_plugin
    return method(*args, **kwargs)
  File "/opt/lib/python3.6/site-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/opt/lib/python3.6/site-packages/flexget/plugins/input/discover.py", line 280, in on_task_input
    entries = self.execute_inputs(config, task)
  File "/opt/lib/python3.6/site-packages/flexget/plugins/input/discover.py", line 109, in execute_inputs
    result = method(task, input_config)
  File "/opt/lib/python3.6/site-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/opt/lib/python3.6/site-packages/flexget/plugins/input/next_series_episodes.py", line 155, in on_task_input
    start_at_ep = max(latest.number - 10, 1)
AttributeError: 'Season' object has no attribute 'number'
2017-09-03 10:07 WARNING  task          download-anime-discover Aborting task (plugin: discover)
2017-09-03 10:07 DEBUG    task_queue                    task download-anime-discover aborted: TaskAbort(reason=BUG: Unhandled error in plugin discover: 'Season' object has no attribute 'number', silent=False)
2017-09-03 10:07 DEBUG    task_queue                    task queue shut down
2017-09-03 10:07 INFO     ipc.rpyc                      server has terminated
2017-09-03 10:07 INFO     ipc.rpyc                      listener closed
2017-09-03 10:07 DEBUG    util.simple_persistence                 Flushing simple persistence for task None to db.
2017-09-03 10:07 DEBUG    manager                       Shutting down
```

#### Crash Log:
```
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Shingeki no Kyojin S2` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Shingeki no Kyojin S2` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Shingeki no Bahamut - Virgin Soul` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Shingeki no Bahamut - Virgin Soul` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Sagrada Reset` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Sagrada Reset` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Re-Creators` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Re-Creators` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Mobile Suit Gundam Unicorn RE 0096` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Mobile Suit Gundam Unicorn RE 0096` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Mobile Suit Gundam - Iron-Blooded Orphans` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Mobile Suit Gundam - Iron-Blooded Orphans` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Macross Delta` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Macross Delta` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Knight's & Magic` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Knight's & Magic` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Hunter X Hunter` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Hunter X Hunter` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Gundam Reconguista in G` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Gundam Reconguista in G` with parameters season: 0, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Granblue Fantasy The Animation` with parameters season: None, downloaded: False
2017-09-03 10:07 DEBUG    series        download-anime-discover no season packs found for series `Granblue Fantasy The Animation` with parameters season: 0, downloaded: False
2017-09-03 10:07 ERROR    series        download-anime-discover cannot compare season when identifier is not 'ep'
2017-09-03 10:07 DEBUG    series        download-anime-discover no episodes found for series `Fate Apocrypha` with parameters season: 11, downloaded: False
2017-09-03 10:07 CRITICAL task          download-anime-discover BUG: Unhandled error in plugin discover: 'Season' object has no attribute 'number'
Traceback (most recent call last):
  File "/opt/lib/python3.6/site-packages/flexget/task.py", line 486, in __run_plugin
    return method(*args, **kwargs)
  File "/opt/lib/python3.6/site-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/opt/lib/python3.6/site-packages/flexget/plugins/input/discover.py", line 280, in on_task_input
    entries = self.execute_inputs(config, task)
  File "/opt/lib/python3.6/site-packages/flexget/plugins/input/discover.py", line 109, in execute_inputs
    result = method(task, input_config)
  File "/opt/lib/python3.6/site-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/opt/lib/python3.6/site-packages/flexget/plugins/input/next_series_episodes.py", line 155, in on_task_input
    start_at_ep = max(latest.number - 10, 1)
AttributeError: 'Season' object has no attribute 'number'
```

### Additional information:

- FlexGet version: 2.10.83
- Python version: 3.6
- Installation method: pip
- Using daemon (yes/no): no
- OS and version: merlin openwrt

**Comment by liiight** [comment]
can you run `flexget series show "Granblue Fantasy The Animation"`?

**Comment by liiight** [comment]
Yeah, i agree. Two underlying issue here, improving the regexp and avoiding the crash.

In the meanwhile you can probably workaround this by doing `flexget series remove "Fate Apocrypha" S11`

**Comment by liiight** [comment]
hmm, recognized correctly for me: 
`Parsing result: <SeriesParser(data=[Ohys-Raws] Fate Apocrypha - 10 (BS11 1280x720 x264 AAC),name=Fate Apocrypha,id=10,season=0,season_pack=None,episode=10,quality=720p h264 aac,proper=0,status=OK)>`

**Comment by programatix** [comment]
This is weird then. It's clearly seen in my log that it took BS11 as Season 11. Anyway, please note the config,
```
    discover:
      what:
        - next_series_episodes:
            backfill: yes
            from_start: yes
```
If `backfill` is `no`, then everything works fine.

**Comment by liiight** [comment]
As long as that entity is registered as a season, flexget will crash since the scenario that a `sequence` series has an episode pack shouldn't exist.
Try looking for the original release title that flexget picked up for that episode using `history` CLI, i wanna reproduce parsing it as a season pack.

**Comment by programatix** [comment]
I run the following command
```flexget history --search 'Fate Apocrypha'```

and I get the following result,
```
╔Showing 19 entries from History════════════════════════════════╗
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_01_[h264-720p][1D074A57].mkv ║
║ URL     ║ https://nyaa.si/download/935321.torrent             ║
║ Time    ║ Thu Aug  3 19:22:31 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_04_[h264-720p][8EE98A4D].mkv ║
║ URL     ║ https://nyaa.si/download/941682.torrent             ║
║ Time    ║ Thu Aug  3 19:53:24 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_03_[h264-720p][DEB2911C].mkv ║
║ URL     ║ https://nyaa.si/download/939468.torrent             ║
║ Time    ║ Thu Aug  3 19:53:24 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_02_[h264-720p][5CC3C3F5].mkv ║
║ URL     ║ https://nyaa.si/download/937228.torrent             ║
║ Time    ║ Thu Aug  3 19:53:24 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_05_[h264-720p][7563C9CB].mkv ║
║ URL     ║ https://nyaa.si/download/944350.torrent             ║
║ Time    ║ Thu Aug  3 19:53:24 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_06_[h264-720p][C7A63CDA].mkv ║
║ URL     ║ https://nyaa.si/download/947461.torrent             ║
║ Time    ║ Tue Aug  8 21:42:06 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_07_[h264-720p][E21730DF].mkv ║
║ URL     ║ https://nyaa.si/download/949228.torrent             ║
║ Time    ║ Sun Aug 13 22:44:42 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_08_[h264-720p][D59A64E0].mkv ║
║ URL     ║ https://nyaa.si/download/951446.torrent             ║
║ Time    ║ Wed Aug 23 17:32:07 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ Anime                                               ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_09_[h264-720p][3CC42AD2].mkv ║
║ URL     ║ https://nyaa.si/download/953568.torrent             ║
║ Time    ║ Thu Aug 31 19:29:20 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_10_[h264-720p][3D8AE10D].mkv ║
║ URL     ║ https://nyaa.si/download/956696.torrent             ║
║ Time    ║ Wed Sep  6 18:07:44 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_11_[h264-720p][B83F914E].mkv ║
║ URL     ║ https://nyaa.si/download/958117.torrent             ║
║ Time    ║ Mon Sep 11 00:07:27 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_12_[h264-720p][B7772227].mkv ║
║ URL     ║ https://nyaa.si/download/959683.torrent             ║
║ Time    ║ Mon Sep 18 04:07:23 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_13_[h264-720p][6B54E188].mkv ║
║ URL     ║ https://nyaa.si/download/964633.torrent             ║
║ Time    ║ Mon Oct  2 10:07:56 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_14_[h264-720p][47B33CCF].mkv ║
║ URL     ║ https://nyaa.si/download/967574.torrent             ║
║ Time    ║ Tue Oct 10 06:20:56 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_15_[h264-720p][47E23E31].mkv ║
║ URL     ║ https://nyaa.si/download/969507.torrent             ║
║ Time    ║ Sat Oct 21 07:23:24 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_16_[h264-720p][C7FFA862].mkv ║
║ URL     ║ https://nyaa.si/download/972264.torrent             ║
║ Time    ║ Tue Oct 24 14:08:39 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_17_[h264-720p][191FECBA].mkv ║
║ URL     ║ https://nyaa.si/download/974612.torrent             ║
║ Time    ║ Wed Nov  1 02:11:52 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_18_[h264-720p][9C146A48].mkv ║
║ URL     ║ https://nyaa.si/download/976381.torrent             ║
║ Time    ║ Mon Nov  6 20:12:47 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
║         ║                                                     ║
║ Task    ║ download-anime-discover                             ║
║ Title   ║ [UTW]_Fate_Apocrypha_-_19_[h264-720p][07204822].mkv ║
║ URL     ║ https://nyaa.si/download/978541.torrent             ║
║ Time    ║ Mon Nov 13 16:16:37 2017                            ║
║ Details ║ Accepted by series (reason: matches quality)        ║
╚═════════╩═════════════════════════════════════════════════════╝
```

**Comment by liiight** [comment]
I dont see `[Ohys-Raws] Fate Apocrypha - 10 (BS11 1280x720 x264 AAC)` in that list

**Comment by programatix** [comment]
Hi,

It looks to me like the 'history' only list out the ones flexget accepted to be downloaded? The '''[Ohys-Raws] Fate Apocrypha - 10 (BS11 1280x720 x264 AAC)''' was never accepted.

**Comment by liiight** [comment]
according to `flexget series show "Fate Apocrypha"` it was, that was the downloaded releases that got parsed as a season pack.

**Comment by programatix** [comment]
There is no ```*``` marking on ```[Ohys-Raws] Fate Apocrypha - 10 (BS11 1280x720 x264 AAC)``` in the ```flexget series show "Fate Apocrypha"``` result.

**Comment by liiight** [comment]
Ah, my bad, misread that table

**Comment by liiight** [comment]
I can't seem to replicate parsing that title as a season pack, not sure how to proceed. Did you try to remove it via `flexget series remove "Fate Apocrypha" S11`?

**Comment by programatix** [comment]
Technically, I have downloaded past episode 11 by changing the configuration as stated in https://github.com/Flexget/Flexget/issues/1956#issuecomment-328282510

So far, this case haven't happened again for other series of anime I'm downloading. It could be that this problem arise due to misconfiguration on my part or a bug in Flexget which has been fixed but due to the data in the database, it occurred. The next time if it arise again, I'll try executing ```flexget series remove "[Anime Name]" Sxx``` to see if it fixes the problem.

I'll just close the case for now. Thanks for the trouble in trying to fix this.
